### PR TITLE
Implement MDServerRemote.FolderNeedsRekey

### PR DESCRIPTION
**Pull request by songgao**
Looks like this needs to go in in order for https://github.com/keybase/client/pull/5521 to pass CircleCI

### CODE REVIEW

**Comment by songgao** [comment]
Sigh; it's failing here now. Perhaps I was suppose to add the `record` first, implement the method in kbfs repo, merge it here, then add the RPC in client ...

**Comment by strib** [comment]
Please remember to assign it to me as well, otherwise I don't see it in the PRs list view.

**Comment by songgao** [comment]
Assigned; sorry for missing that in the first place!

File: libkbfs/mdserver_remote.go [hunk 1]
```diff
@@ -570,8 +570,31 @@ func (md *MDServerRemote) MetadataUpdate(_ context.Context, arg keybase1.Metadat
 	return nil
 }
 
+// FoldersNeedRekey implements the MetadataUpdateProtocol interface.
+func (md *MDServerRemote) FoldersNeedRekey(_ context.Context,
+	requests []keybase1.RekeyRequest) error {
+	if md.squelchRekey {
+		md.log.Debug("MDServerRemote: rekey updates squelched for testing")
```
Comment (strib) [inline libkbfs/mdserver_remote.go]: Please use `CDebugf(ctx,` here and below (`FolderNeedsRekey` should do that too, but you don't need to fix it in this PR).

**Comment by songgao** [comment]
@strib hmm so I'll revert the last commit in https://github.com/keybase/client/pull/5521 (i.e. add back the RPC), merge it, and merge this with updated vendor?

**Comment by songgao** [comment]
nvm just saw your comment there!

**Pull request merged**